gcc/config/riscv/predicates.md

```diff
@@ -236,11 +236,15 @@
 
 (define_predicate "mem_post_inc"
   (and (match_code "mem")
-       (match_test "TARGET_XCVMEM && GET_CODE (XEXP (op, 0)) == POST_MODIFY")))
+       (match_test "TARGET_XCVMEM && GET_CODE (XEXP (op, 0)) == POST_MODIFY
+		    && GET_MODE (op) != DImode
+                    && GET_MODE (op) != DFmode")))
```

Comment:
Is it DI / DF specifically that you want to exclude or modes larger than a certain size? What about TI / TF for example?

Reply:
Good idea